### DISCUSSION TITLE
Contribute 2.0

### DISCUSSION THREAD

**Discussion by HectorSancho** [Telefonica/mistica-design#1555] [Proposals]
### Goal

Mejorar el modelo de contribución actual.

Ejemplos:

https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github
https://carbondesignsystem.com/contributing/get-started/

### Context

Repensar la contribución a Mística por parte de los equipos. También valorar como poner a disposición de los usuarios los componentes creado por otros equipos.

### Impact

Conseguir que los equipos externos contribuyan más a Mística y que todos los usuarios del sistema se puedan beneficiar de ese trabajo.

### Discussion Questions

- ¿Nuevo modelo de contribución?
- ¿Varios modelos?
- ¿Cómo se hacen disponibles los componentes de unos equipos a otros?
- ¿Qué nivel de definción y/o desarrollo debe tener el componente para ser compartido?

**Comment by AnaMontes11** [comment]
Pego por aquí las secciones "[Working with us](https://paste.twilio.design/introduction/working-with-us)" y "[Contributing](https://paste.twilio.design/introduction/contributing/components)" de Twilio ya que me ha parecido interesante el enfoque.

**Reply by AnaMontes11** [reply]
Decathlon: 
- https://www.decathlon.design/726f8c765/p/195920-contributing/b/04a4ff
- https://www.decathlon.design/726f8c765/p/195920-contributing/b/14d247
- https://www.decathlon.design/726f8c765/p/24ef08-contributing/b/237af4
